community/blog/2021 Quick overview at the new major Arcus Messaging v1.0 release/README.md

```diff
@@ -0,0 +1,144 @@
+# Quick overview at the new major Arcus Messaging v1.0 release
+The new major v1.0 Arcus Messaging release is the biggest one yet: internal improvments, authentication updates, new packages, and many more.
+
+Because the new version contains a lot of breaking changes, we've created a [dedicated migration guide](https://messaging.arcus-azure.net/guides/migration-guide-v1.0) to help consumers in updating towards this new major version.
+
+Come and take a quick look what the release has to offer in this overview blog post.
+
+## New Azure SDK
+Microsoft released a [new client library SDK](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/README.md) for interacting with Azure Service Bus resources. This new library is part of the overall Azure SDK updates. Our [Arcus Security](https://github.com/arcus-azure/arcus.security) library [is already updated](https://github.com/arcus-azure/arcus.security/issues/191) to use this new Azure SDK, so now it was time to update Arcus Messaging.
+
+The most notable change here for consumers is the move from the Azure Service Bus `Message` model towards the `ServiceBusReceivedMessage`. This breaking change has the most impact on our fallback message handlers, which receive the full Azure Service Bus message in their implementation.
+
+An example of this new signature:
+
+```csharp
+using Arcus.Message.Abstractions.ServiceBus.MessageHandling;
+using Azure.Messaging.ServiceBus;
+
+public class OrderFallbackMessageHandler : IAzureServiceBusFallbackMessageHandler
+{
+    private readonly ILogger _logger;
+
+    public OrderFallbackMessageHandler(ILogger<OrderFallbackMessageHandler> logger)
+    {
+        _logger = logger;
+    }
+
+    public Task ProcessMessageAsync(
+        ServiceBusReceivedMessage message,
+        AzureServiceBusMessageContext messageContext,
+        MessageCorrelationInfo messageCorrelationInfo,
+        CancellationToken cancellactionToken)
+    {
+        _logger.LogInformation("Start processing Azure Service Bus message {MessageId}...", message.MessageId);
+    }
+}
+```
+
+Microsoft has also provided a [dedicated migration guide](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md) for this new Azure SDK if you're curious in any new funtionality or changes.
+
+## Extracted message routing for multi-purpose message handling
+Versions before the v1.0 release all used the message pump to both receive and route the messages to the correct registered message handler. In this release, we have made sure that these two resposibilities are split between two services. The message pump still receives the messages, but now a dedicated router will determine and delegate the message to the message handler that can handle the message.
```

Comment:
What do you think of this? It was a bit much/not fully clear imo
```suggestion
Before v1.0, our message pump was in charge of both receiving and routing messages to the required message handler. 

With this release, we are separating these two responsibilities into dedicated components and laying the groundwork for more powerful message processing.

The message pump is still in charge of receiving messages, but relies on our message router to determine what message handler will process the message.
```

Reply:
Is great! Bit less technical and more functional which makes this more readable, indeed. Thanks!